lib/Parse/ParseStmt.cpp

```diff
@@ -291,7 +292,7 @@ ParserStatus Parser::parseBraceItems(SmallVectorImpl<ASTNode> &Entries,
     PreviousHadSemi = false;
     if (isStartOfDecl()
         && Tok.isNot(tok::pound_if)
-        && Tok.isNot(tok::pound_line)) {
+        && Tok.isNot(tok::pound_setline)) {
```

Comment:
Minor nitpick, you can use Tok.isNot(x, y) here, though it looks like a preexisting issue.